Issue with starting test server

Hello

When I go to run "npm start" on the test server I get this error message
`C:\Users\Brandon\colyseus-construct-plugin\test\server>npm start

> server-colyseus@1.0.0 start C:\Users\Brandon\colyseus-construct-plugin\test\server
> DEBUG=* ./node_modules/.bin/supervisor -i node_modules -- index.js

'DEBUG' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! server-colyseus@1.0.0 start: `DEBUG=* ./node_modules/.bin/supervisor -i node_modules -- index.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the server-colyseus@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Brandon\AppData\Roaming\npm-cache\_logs\2017-05-30T20_36_12_119Z-debug.log`

Would you know why I may be getting this?

Thank you @endel 

When I remove the environment variable it is still throwing me this error:

`Error: Cannot find module 'colyseus'
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.Module._load (module.js:437:25)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Users\Brandon\colyseus-construct-plugin\test\server\index.js:3:16)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
Program node index.js exited with code 1`

--- a/test/server/package.json
+++ b/test/server/package.json
@@ -4,7 +4,7 @@
   "description": "Simple game server with Colyseus framework",
   "main": "index.js",
   "scripts": {
-    "start": "DEBUG=* ./node_modules/.bin/supervisor -i node_modules -- index.js"
+    "start": "cross-env DEBUG=* ./node_modules/.bin/supervisor -i node_modules -- index.js"
   },
   "author": "Loïc Guglielmino",
   "license": "MIT",
@@ -13,6 +13,7 @@
   },
   "devDependencies": {
     "supervisor": "^0.12.0",
-    "debug-logger": "^0.4.1"
+    "debug-logger": "^0.4.1",
+    "cross-env": "^5.0.0"
   }
 }

